routes/notes.router.js

```diff
@@ -0,0 +1,86 @@
+const express = require('express');
+const router = express.Router();
+const { Note } = require('../models/note.model');
+
+router.get('/', async (req, res) => {
+    res.json('you are here brah');
+});
+
+//getting all notes for a user for a video
+//req.body:{videoId}
```

Comment:
If you're writing comments as a part of your documentation, you could look at some libraries which would generate complete webpages from your commented documentation.

Reply:
Good idea, I'll surely look it up.